test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -1223,32 +1219,6 @@ def closure_ddp():
                         atol=1e-04,
                     ), "Models differ after a step"
 
-    @common_distributed.skip_if_lt_x_gpu(4)
-    @parametrize(
-        "parameters_as_bucket_view",
-        [False, True],
-    )
-    def test_zero_model_parallel(
```

Comment:
shouldn't we just remove the replicated tensor enable / disable, but keep the test (i.e. still call `_test_zero_model_parallel`), instead of removing the test completely?

cc @awgu who originally wrote this test.

Reply:
Oh I might got the meaning of "Model Parallel" wrong here. I will change back.